selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/DashboardFactories.java

```diff
@@ -186,10 +192,8 @@ public void clickOnAddFactoryBtn() {
   public void selectWorkspaceForCreation(String wsNama) {
     // delay for animation page
     WaitUtils.sleepQuietly(1, TimeUnit.SECONDS);
```

Comment:
It's better to use dedicated class **TestWebElementRenderChecker** to wait animation is ended.